src/tools/html-checker/main.rs

```diff
@@ -0,0 +1,65 @@
+use std::env;
+use std::path::Path;
+use std::process::{Command, Output};
+
+fn check_html_file(file: &Path) -> bool {
```

Comment:
Running this once per file will be slow (rustdoc generates *lots* of files). Could you run it on all files at once instead and see how the time compares?

Reply:
I don't know how well the system calls to start processes will appreciate thousands of arguments...